### PR TITLE
CMR-7020 - adding a clarification on when tests are needed

### PR DESCRIPTION
**Pull request by jceaser**
for ticket #10, clarify the rules on when tests are needed

The Contribution page has been updated.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,10 +8,19 @@ To allow us to incorporate your changes, please use the following process:
...
-3. Test the changes locally/in your personal fork.
-4. Submit a pull request to open a discussion about your proposed changes.
-5. The maintainers will talk with you about it and decide to merge or request additional changes.
+3. Create a branch and make your changes, including tests to verify your change
+4. If applicable, test the changes locally/in your personal fork. See section below.
+5. Submit a pull request to open a discussion about your proposed changes.
+6. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+### Tests
+Not all code requires tests. Code meant as example, documentation, or experimental in nature do not need to have matching tests. Code meant to be authoritative or operational however must. Examples types of code that require tests are:
```
Comment (jaybarra) [inline CONTRIBUTING.md:17]: Grammar fix: "in nature does not need"

**Review by ygliuvt — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jceaser) [reply CONTRIBUTING.md:17]: jedi in grammar, you will be

**Review by mmorahan — APPROVED** [review]

**Review by sritz-23 — APPROVED** [review]
Looks good.

**Comment by jceaser** [comment]
Included in this change is a suggestion from Joe who also reviewed the code change. This was the last commit made, his recommendation.

**Pull request merged**